src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataRead = readFromFile(fromFileName);
+        StringBuilder sumData = sumData(dataRead);
+        File file = new File(toFileName);
+        BufferedWriter writer = null;
+        try {
+            writer = new BufferedWriter(new FileWriter(file, true));
+            writer.write(String.valueOf(sumData));
+            writer.flush();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file", e);
+        } finally {
+            try {
+                writer.close();
+            } catch (IOException e) {
+                throw new RuntimeException("Can't write file");
+            }
+        }
```

Comment:
move it to a separate method also, and use try-with-resources

Reply:
https://github.com/mate-academy/jv-oop/pull/1054#pullrequestreview-1163912945

Reply:
Create method for write report.